libexec/rtld-elf/rtld_c18n.h

```diff
@@ -193,6 +193,19 @@ pop_dummy_rtld_trusted_frame(struct trusted_frame *tf)
 /*
  * Stack unwinding
  */
+#ifdef rtld_get_return_address
+#undef rtld_get_return_address
```

Comment:
The #undef seems a bit fragile.  Can we use an #ifdef in rtld.h so that it is just always defined correctly?